user_docs/en/userGuide.t2t

```diff
@@ -1074,6 +1074,27 @@ When you scroll the display, you will then be able to read the rest of the word.
 
 Enabling this may allow for more fluent reading, but generally requires you to scroll the display more.
 
+==== Focus context presentation ====
+This option allows you to choose what context information NVDA will show on the braille display when an object gets focus.
+Context information (or focus ancestry) refers to the hierarchical tree of objects containing the focus object.
+For example, when you just focused a list item, this list item is part of a list. This list might be contained by a dialog, etc.

```

Comment:
1. "when you just focused" -> "when you focus"
2. Two sentences on one line. Please split.